index.js

```diff
@@ -1461,6 +1461,22 @@ class Command extends EventEmitter {
     return width;
   };
 
+  /**
+   * Convert value to a string for help
+   *
+   * @param {object} value
+   * @return {string}
+   */
+
+  _stringify(value) {
+    // Use the `toString` method for objects that don't use the default
+    if (typeof value === 'object' && Object.getPrototypeOf(value).toString !== Object.prototype.toString) {
```

Comment:
Turns out `JSON.stringify` is very robust! When I tried putting in some challenging default values, the new code crashed displaying the help. e.g.

```
.option('--null <value>', 'null', null)
.option('--cleanObject <value>', 'Object.create(null)', Object.create(null))
```

I did a bit of reading and experimentation and currently testing:

```
    if (typeof value === 'object' && value !== null && typeof value.toString === 'function' && value.toString !== Object.prototype.toString) {
```